src/api.js

```diff
@@ -20,4 +20,12 @@ export const getComments = (review_id) => {
     return houseOfGamesApi.get(`/reviews/${review_id}/comments`).then((res) => {
         return res.data.comments
     })
+}
+
+export const patchVotesReview = (review_id, vote) => {
+    const patchBody = { inc_votes: vote }
```

Comment:
Really like you've used a separate variable to store the body, it makes the code super readable 👍